build.gradle

```diff
@@ -18,16 +18,14 @@ buildscript {
     }
     dependencies {
         classpath "biz.aQute.bnd:biz.aQute.bnd.gradle:5.3.0"
-        classpath("com.github.jruby-gradle:jruby-gradle-plugin:2.0.1") {
-          exclude module: 'grolifant'
-          exclude module: 'okhttp-digest'
-        }
-        classpath 'org.ysb33r.gradle:grolifant:0.17.0'
     }
 }
 
 // modern plugins config
 plugins {
+  id 'org.ysb33r.jruby.resolver'    version '2.0.0-alpha.4'
+  id 'org.ysb33r.jruby.base' version '2.0.0-alpha.4'
```

Comment:
The final the `2.0.0` was released today.

Reply:
Thanks! Updated to 2.0.0 and the tests seem to pass.